lib/project/results-pane.coffee

```diff
@@ -140,3 +149,19 @@ class ResultsPaneView extends ScrollView
       @removeClass('no-results')
     else
       @addClass('no-results')
+
+  collapseAllResults: =>
+    i=0
```

Comment:
will make changes. Also, does it make sense to instead making the logic here or extend the `collapseResult` and `expandResult` in the `results-view.coffee`?